test/unit/storage_test.rb

```diff
@@ -113,6 +113,35 @@ def test_sentinels_simple_url
                                       sentinels: [{ host: '127.0.0.1', port: 26_379 }])
   end
 
+  def test_sentinels_array_hashes_default_port
+    default_sentinel_port = Storage::Helpers.singleton_class.const_get(:DEFAULT_SENTINEL_PORT)
+    config_obj = {
+      url: 'redis://127.0.0.1:6379/0',
+      sentinels: [{ host: '127.0.0.1' }, { host: '192.168.1.1'},
+                  { host: '192.168.1.2', port: nil }]
```

Comment:
No, because this is already tested in another test methods like 'test_sentinels_connection_array_hashes', 'test_sentinels_malformed_url', 'test_sentinels_connection_array_strings, etc...

The idea of this specific test is to test sentinels without ports

Reply:
the combination of both things at the same time is not tested, no? :) At the very least _some_ test should do it.

Reply:
I agree that it is a good idea to test the combination of both things at the same time is a good idea to test so I have added it :+1:

If it was not for the possibility of the combination I would not think it was necessary to add an actual port in the test because it is tested in other tests already and I don't see the value on adding it in yet another test where is covered in other already existing tests.